ohsome/clients.py

```diff
@@ -142,12 +142,8 @@ def start_timestamp(self):
         Returns the temporal extent of the current ohsome API
         :return:
         """
-        if self._metadata is None:
-            self._query_metadata()
         return dt.datetime.fromisoformat(
-            self._metadata["extractRegion"]["temporalExtent"]["fromTimestamp"].strip(
-                "Z"
-            )
+            self.metadata["extractRegion"]["temporalExtent"]["fromTimestamp"].strip("Z")
```

Comment:
Consider using `@cached_property` for `metadata`. It can let you simplify the way how you set the property.

Reply:
done